tests/protractor/helper.js

```diff
@@ -1,38 +1,58 @@
 const fs = require('fs'),
-      EC = protractor.ExpectedConditions;
+  EC = protractor.ExpectedConditions;
 
 function writeScreenShot(data, filename) {
   const stream = fs.createWriteStream(filename);
   stream.write(new Buffer(data, 'base64'));
   stream.end();
 }
+function retryFind(locator) {
+  let result = false;
+  let attempts = 0;
+  while (attempts < 2) {
+    try {
+      browser.actions().mouseMove(locator).perform();
+      result = true;
+      break;
+    } catch (err) {
+      browser.sleep(200);
+      console.log(err);
+    }
+    attempts++;
+  }
+  return result;
+}
 
 module.exports = {
-  waitElementToBeVisisble: elm=> {
+  waitElementToBeVisisble: elm => {
     browser.wait(EC.visibilityOf(elm), 15000);
   },
 
-  waitElementToBeClickable: elm=> {
+  waitElementToBeClickable: elm => {
     browser.wait(EC.elementToBeClickable(elm), 15000);
   },
 
-  waitElementToDisappear: locator=> {
-    browser.wait(()=> {
+  waitElementToDisappear: locator => {
+    browser.wait(() => {
       return element(locator).isPresent()
-        .then(presenceOfElement=> {
+        .then(presenceOfElement => {
           return !presenceOfElement;
         });
     }, 10000);
   },
+  waitElementToBeReattached: locator => {
+    //return browser.wait(() => { return retryFind(locator); }, 10000);
+    return retryFind(locator);
```

Comment:
If all this function does is call `retryFind` then what's the point of it? Can't we just remove it and use `retryFind` instead?